examples/subscribing-example-app/public/style.css

```diff
@@ -207,6 +207,8 @@ p {
 
 .hidden {
   visibility: hidden;
+  display: none !important;
+  margin-top: 0 !important;
```

Comment:
Why is the margin changed here, was there a layout issue just using `display: none`?

Reply:
I think this is just an artefact of me trying to fix a layout issue but yeah you're right, the `margin-top` wasn't doing anything. I've removed it in a4ff916.